polyfill-service-api/polyfills/Object/getPrototypeOf/polyfill.js

```diff
@@ -1,7 +1,11 @@
-Object.getPrototypeOf = function getPrototypeOf(object) {
-	if (object !== Object(object)) {
-		throw new TypeError('Object.getPrototypeOf called on non-object');
+(function(_getPrototypeOf, Object) {
+	Object.getPrototypeOf = function getPrototypeOf(o) {
+		if (o === null || o === undefined) {
```

Comment:
this probably can be changed to `if (o == null) {` which is effectively the same.